etna/records/widgets.py

```diff
@@ -30,7 +30,7 @@ def get_instance(self, pk):
         """Fetch related instance on edit form."""
         try:
             return Record.search.get(iaid=pk)
-        except (KongAPIError, SearchManagerException):
+        except (KongAPIError, APIManagerException):
             # If there's a connection issue with Kong, return a stub Record
             # so we have something to render on the ResultsPage edit form.
             return Record(iaid=pk, title="", reference_number="")
```

Comment:
- Why do we return a stub record to render something when its an error. Should we not have an error page so that it must be rectified - may be some alert?
- Should these error be handled separately?

Reply:
Moving conversation to https://github.com/nationalarchives/ds-wagtail/pull/234#pullrequestreview-887408124